test/best.js

```diff
@@ -59,10 +59,10 @@ test('pre-cut off number', (t) => {
   t.plan(1)
   const actual = convert(9000).from('mm').toBest({ cutOffNumber: 10 })
   const expected = {
-    val: 900,
-    unit: 'cm',
-    singular: 'Centimeter',
-    plural: 'Centimeters'
+    val: 90,
```

Comment:
Same here: could you update just the `cutOffNumber` to `100` to get the same expected response?